data/scenarios/Challenges/2048.yaml

```diff
@@ -0,0 +1,188 @@
+name: "2048"
+description: Make 2048!
+objectives:
+  - goal:
+    - OK, it's not really the same as the classic "2048" game.
+      However, your goal is still to make 2048.
```

Comment:
This description is technically all you need, but maybe it could be made more exciting. :grinning: 

The "make 2048" could also be expanded upon. In my experience, since you do not see under the robot it is not immediately obvious what you should do with the `1`. IMO, it should be noted that the `1` are harvestable.